src/gui/sdlmain.cpp

```diff
@@ -4118,6 +4118,80 @@ static std::vector<std::string> get_sdl_texture_renderers()
+	        "\n"
+	        "  --eraseconf              Delete the primary configuration file.\n"
+	        "\n"
+	        "  --noprimaryconf          Don't load or create the the primary configuration file.\n"
+	        "\n"
+	        "  --nolocalconf            Don't load the local 'dosbox.conf' configuration file\n"
+	        "                           if present in the current working directory.\n"
+	        "\n"
+	        "  --set <setting>=<value>  Set a configuration setting. Multiple configuration\n"
+	        "                           settings can be specified. Example:\n"
+	        "                           --set mididevice=fluidsynth --set soundfont=mysoundfont.sf2\n"
+	        "\n"
+	        "  --working-dir <path>     Set working directory to <path>. DOSBox will act as if\n"
+	        "                           started from this directory.\n"
+	        "\n"
+	        "  --list-glshaders         List all available OpenGL shaders and their paths.\n"
+	        "                           Results are useable in the 'glshader = ' config setting.\n"
+	        "\n"
+	        "  --fullscreen             Start in fullscreen mode.\n"
+	        "\n"
+	        "  --lang <lang_file>       Start with the language specified in <lang_file>.\n"
+	        "\n"
+	        "  --machine <type>         Emulate a specific type of machine. The machine type has\n"
+	        "                           influence on both the emulated video and sound cards.\n"
+	        "                           Valid choices are: hercules, cga, cga_mono, tandy,\n"
+	        "                           pcjr, ega, svga_s3 (default), svga_et3000, svga_et4000,\n"
+	        "                           svga_paradise, vesa_nolfb, vesa_oldvbe.\n"
+	        "\n"
+	        "  -c <command>             Run the specified DOS command before running FILE.\n"
+	        "                           Multiple commands can be specified.\n"
+	        "\n"
+	        "  --noautoexec             Don't execute DOS commands from any [autoexec] sections.\n"
+	        "\n"
+	        "  --exit                   Exit after the DOS program specified by FILE has ended.\n"
+	        "                           If no FILE has been specified, execute [autoexec] and\n"
+	        "                           terminate.\n"
+	        "\n"
+	        "  --startmapper            Run the mapper GUI.\n"
+	        "\n"
+	        "  --erasemapper            Delete the default mapper file.\n"
+	        "\n"
+	        "  --securemode             Enable secure mode: disables the MOUNT and IMGMOUNT commands.\n"
+	        "\n"
+	        "  --socket <num>           Run nullmodem on the specified socket number.\n"
+	        "\n"
+	        "  -h, --help               Print this help message and exit.\n"
+	        "\n"
+	        "  -v, --version            Print version information and exit.\n"
+	        "\n"
+	        "FILE                       A mountable path such as a directory, drive image\n"
```

Comment:
Isn't it customary to put the description of such FILE / PATH / DIR arguments at the top of the help text, before the detailing the various options?

Reply:
yup; moved to the top.